docs/content/guides/migrations/coexistence-of-airflow-and-dagster.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: "Coexistence of Airflow and Dagster | Dagster Docs"
+description: "Learn how to leverage the features of Dagster and Airflow together."
+---
+
+# Coexistence of Airflow and Dagster
```

Comment:
This is very much still a draft, and could use some feedback on general naming. This guide does not need to be released with the other PRs, and can wait until a more formal release.

But I figured it would be insightful to show the draft.